SS14.Launcher/LauncherCommands.cs

```diff
@@ -0,0 +1,144 @@
+using System;
+using System.Buffers;
+using System.Diagnostics;
+using System.IO;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Avalonia.Threading;
+using Serilog;
+using SS14.Launcher.ViewModels;
+using SS14.Launcher.Models.Logins;
+
+namespace SS14.Launcher;
+
+public static class LauncherCommands
+{
+    private static IDisposable? _timer;
+    private static string _reason = "";
+
+    /// <summary>
+    /// Starts the timer that handles commands and either defers them (connection commands when not ready) or handles them.
+    /// </summary>
+    public static void StartReceivingTimer(MainWindowViewModel windowVm, LoginManager loginMgr)
+    {
+        _timer = DispatcherTimer.Run(() =>
```

Comment:
Don't use a timer for this. Post to the UI thread via `Dispatcher.UIThread` and either have the pipe thread do the command parsing itself or use a `Channel` of some kind for it (the former reduces complexity and is probably fine).

Reply:
Not using a separate thread, so it's all fine now